code/topology/README.md

```diff
@@ -10,3 +10,4 @@
 ## Usage
 
     ./hco_topo.sh | dot -Tpng -o topo.png
+
```

Comment:
Not sure if this will work, may have to checkout this file from another branch to undo the trailing newline.
```suggestion
```

Reply:
Done